connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/DistributedHerderTest.java

```diff
@@ -935,9 +936,6 @@ public Object answer() throws Throwable {
         herder.tick();
         assertTrue(connectorConfigCb.isDone());
         assertEquals(CONN1_CONFIG_UPDATED, connectorConfigCb.get());
-        // The config passed to Worker should
```

Comment:
True, they only matter during validation & task configuration (or really any time the config is handed over to the connector). I think its fine to use them only in those cases, I was just losing track of where in the code it mattered and where it didn't.